mmv1/products/dns/Policy.yaml

```
@@ -50,6 +53,12 @@ examples:
       org_id: 'ORG_ID'
       billing_account: 'BILLING_ACCT'
     exclude_docs: true
+virtual_fields:
+  - name: 'enableDns64AllQueries'
```

```suggestion
  - name: 'enable_dns64_all_queries'
```

since this api doesn't have a field of this name we can use the terraform name here